### ISSUE TITLE
Clicking directly on text within a text box does not place the text cursor as of 1.15.1

### ISSUE THREAD

**Issue by adamdunnage** [swissspidy/web-stories-wp#10148]
## Bug Description

When directly clicking or double clicking on text within a text box while editing a story the text cursor is not placed where clicked. If you do click underneath the text where you wish to place the cursor then this will work but not when directly clicking on the text.

This only happens in the latest 1.15.1 version and before this you can directly click on the text and the cursor will be placed where clicked.

**See here what happens when clicking directly on the text in 1.15.1:** 
![WS-Text-Doubleclick](https://user-images.githubusercontent.com/91896826/148910019-f6eae185-59a3-4cb9-aa5b-be74ac34780a.gif)

**See what happens when clicking below the text in 1.15.1:**
![WS-text-clickbelow](https://user-images.githubusercontent.com/91896826/148910271-f3840b0d-3524-4a1d-8070-3c87e505a68a.gif)

**What happens when clicking directly on the text in 1.15:**
![WS-TextClick-previous-version](https://user-images.githubusercontent.com/91896826/148912823-1988206a-d7bc-474c-bce0-ae855253a5ec.gif)

## Expected Behaviour

I would expect when clicking directly on the text after the box has been selected or double clicking would place the text cursor where clicked as it does in the previous plugin version. I wouldn't expect to have to click below the text as this is not very user friendly at all and would be missed by many users.

## Steps to Reproduce

1. Create or edit a Story.
2. Place text on a story page.
3. Double click directly on the text in the middle of a word or single click if the text box has been selected.

## Screenshots

GIFs provided above.

**Comment by swissspidy** [comment]
Just heard something similar yesterday internally. Let‘s prioritize this.

**Comment by merapi** [comment]
First I've tried to replicate this on macOS/Windows with Chrome/Safari/Firefox with no luck (on main). Then went back to 1.15.0 and 1.15.1 and indeed it occurred only on 1.15.1 and was later fixed 👍🏻 

Broken in cf2eece3045e7decc2b5cdff1a3f2b4e6c3c71f7
Fixed in 3d2e7f2dc435b9734e3eb2493c87d43dbb6e9bfb

**Comment by swissspidy** [comment]
Oh so this is fixed on main?

**Comment by merapi** [comment]
Yes, should we just now move it to QA (for double-checking)?

**Comment by miina** [comment]
> Yes, should we just now move it to QA (for double-checking)?

Sounds good, maybe leave some extra testing instructions for @felipebochehin87 as well.

**Comment by merapi** [comment]
@felipebochehin87 I think there is no point in testing the bad behavior in 1.15.1 as we have clear instructions and gifs on how it looked, just test the "Expected Behaviour" on the current version and we are good to go.

**Comment by felipebochehin87** [comment]
Tested against https://stories-qa-wordpress-amp.pantheonsite.io/wp-admin/, using Main branch.

1. Added a text box
2. Single clicked - text box selected
3. Clicked again - cursor positioned where I clicked, enabling text for editing
4. Clicked out of the canvas
5. Double clicked text box - cursor positioned where I clicked, enabling text for editing


https://user-images.githubusercontent.com/96055551/149541193-358dda37-8391-4c8f-a1b5-b84a1eca1914.mp4
